.github/CHANGELOG.md

```diff
@@ -7,4 +7,8 @@
 
 1. [General] Add CHANGELOG.md - @nathaninnes (Nathan Innes)
 2. [ECAM] Improved Upper ECAM Visuals - @wpine215 (Iceman)
+1. [CDU] Corrected MCDU flight phase - @aryeng (ryeng)
+<<<<<<< HEAD
 
+=======
+>>>>>>> 368676ef45c529b456b8fe077eea78bfde505dc4
```

Comment:
You have unresolved merge conflicts in this file.

Reply:
I commited your suggested, however still conflicting. Nathan told me yesterday to not worry about the conflict in changelog.

Reply:
Updated the changelog. Seems to have cleared the condlict.